.buildkite/publish/build-docker.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -exu
+set -o pipefail
+
+function realpath {
+  echo "$(cd "$(dirname "$1")"; pwd)"/"$(basename "$1")";
+}
+
+SCRIPT_DIR=$(realpath "$(dirname "$0")")
+BUILDKITE_DIR=$(realpath "$(dirname "$SCRIPT_DIR")")
+PROJECT_ROOT=$(realpath "$(dirname "$BUILDKITE_DIR")")
+
+if [[ "${ARCHITECTURE:-}" == "" ]]; then
+  echo "!! ARCHITECTURE is not set. Exiting."
+  exit 2
+fi
+
+VERSION_PATH="$PROJECT_ROOT/connectors/VERSION"
+VERSION=$(cat $VERSION_PATH)
+
+if [[ "${USE_SNAPSHOT:-}" == "true" ]]; then
```

Comment:
I don't see anything that ever sets this. Is this just for if you're running locally, instead of through buildkite?

Reply:
In Buildkite, you can add add environment variables when you launch a build - I figured it was easy enough to at least add the option in...

![image](https://github.com/elastic/connectors/assets/13320310/1e4e2457-ec32-4a83-b859-feb3ad4d4bef)

Reply:
Oh sweet! TIL. I was sure there was a way to create special build params, but didn't realize it was this easy to add ad-hoc env vars.